screens/PokemonFromTheList.tsx

```diff
@@ -42,7 +42,7 @@ export default function PokemonListMoreInfo({ route, navigation }: Props) {
   const onPress = useCallback(() => {
     setIconName(heartFull);
```

Comment:
if you have constants (like `const heartEmpty = 'md-heart-empty';` which is always the same) it's usually better to put the above the function (`PokemonListMoreInfo`) like this: `const HEART_EMPTY_ICON = 'md-heart-empty';`, otherwise you will initialize this variable in every function call (every render)